test/jdk/sun/security/provider/SecureRandom/AutoReseed.java

```diff
@@ -28,7 +28,7 @@
  * @test
  * @bug 8051408
  * @summary make sure nextBytes etc can be called before setSeed
- * @run main/othervm -Djava.security.egd=file:/dev/urandom AutoReseed
```

Comment:
This test uses generateSeed and needs egd to avoid reading from /dev/random

Reply:
Backing out change.